types/email-templates/index.d.ts

```diff
@@ -140,6 +147,11 @@ interface EmailConfig<T = any> {
      * <https://github.com/Automattic/juice>
      */
     juiceResources?: any;
+    /**
+     * a function that returns the path to a template file
+     * @default (path: string, template: string) => string
+     */
+    getPath?: (path: string, template: string, locals: any) => string;
```

Comment:
I' don't see locals as a variable as an input for the config, 
```typescript
getPath(type, template) {
   return path.join(template, type);
}

Reply:
@blankstar85 done, amended. Thx for review!

Reply:
Sorry I mean to say this input getPath in the config, doesn't have or require a locals.
getpath?: (path: string, template: string) => string;

Reply:
this default one, yes:
https://github.com/forwardemail/email-templates/blob/master/src/index.js#L103
But is being called with locales:
https://github.com/forwardemail/email-templates/blob/master/src/index.js#L162
(as per readme:
https://github.com/forwardemail/email-templates/blob/master/README.md#L705

Reply:
I overlooked the doc. Your right, you wouldn't change it otherwise. Thanks!